src/.nuget/dir.targets

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+
+  <Target Name="VerifyVCRedist" BeforeTargets="GetSymbolPackageFiles" Condition="'$(_runtimeOSFamily)' == 'win'">
```

Comment:
Yes, it's just changing the model slightly.  They still need to be picked up from somewhere for packaging/publishing.  You don't have to produce that package with every build though and you could just consume the package in coreclr rather than packaging the build machine state.  Somebody has to have a machine requirement on the binaries, but it doesn't have to be CoreClr.  The ApiSet packages used to be produced out of ProjectK, I believe.  CoreFx has moved away from using the ApiSet packages, but I think that's more an evolution of the product than an issue with the packages.  I don't know enough about the history to really fight for it being a better model, I'm just suggesting it as a possible alternative.